launcher_scripts/conf/config.yaml

```diff
@@ -57,6 +57,8 @@ wandb_api_bcp_secret_key: null  # For BCP clusters, read the W&B api key directl
 
 bcp_no_redirect: True  # If True, all stdout and stderr will not be redirected and appear in the standard logs. If False, all stdout and stderr output will be redirected to individual files on a per-rank basis. Ignored for non-BCP clusters.
 
+enable_vboost: False # If True, enables video boost slider- benefits compute bound jobs
```

Comment:
Let's update the comment as "If True, rebalance the power budget between the compute engine and the memory such that the compute engines has greater power allocation"